docs/modular-resources/advanced/mlmodel-design.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Design your ML Models for Vision"
+linkTitle: "Vision Service and Modular ML Model Design"
+weight: 60
+type: "docs"
+tags: ["data management", "ml", "model training"]
+description: "Design your ML Model service to work with Viam's vision services."
+icon: "/services/icons/ml.svg"
+# SME: Bijan Haney
+---
+
+Models of Viam's [Machine Learning (ML) model service](/services/ml/) allow you to deploy machine learning models to your smart machine.
```

Comment:
While this is technically correct, I think in this case it's ok to just speak of the service in the introduction
```suggestion
The [Machine Learning (ML) model service](/services/ml/) allow you to deploy machine learning models to your smart machine.
```

Reply:
Sounds good